core/src/main/java/io/questdb/cairo/wal/seq/SeqTxnTracker.java

```diff
@@ -136,4 +224,13 @@ public void setUnsuspended() {
         this.errorTag = ErrorTag.NONE;
         this.errorMessage = "";
     }
+
+    public boolean shouldBackOff(long nowMicros) {
```

Comment:
A comment why it should back off would be good or indicate in the method name that it's memory related.

Reply:
Renamed to `shouldBackOffDueToMemoryPressure()`